docs/source/getting_started/datasets.rst

```diff
@@ -26,8 +26,55 @@ Datasets can be listed and gotten from projects just as projects are from the cl
 Datasets can only be deleted if they are empty.
 If you need to delete a non-empty dataset, see this discussion on :ref:`deleting data objects <deleting_data_objects_label>`.
 
-Dataset Access and Sharing
---------------------------
+Dataset Creation and Deletion
+-----------------------------
+A dataset can be created (or deleted) using the project from which it should or currently originates using the :func:`~citrine.resources.dataset.DatasetCollection.register` and :func:`~citrine.resources.project.ProjectCollection.delete` methods. Note that the creation of a dataset requires the creation of a `Dataset` object.
```

Comment:
```suggestion
A dataset can be created (or deleted) using the project from which it should or currently originates using the :func:`~citrine.resources.dataset.DatasetCollection.register` and :func:`~citrine.resources.dataset.DatasetCollection.delete` methods.
```
Based on the code example below it looks like the method is `project.datasets.delete`, not `citrine.projects.delete`.

Reply:
What does "from which it should or currently originates" mean?

Reply:
I suggested deleting the second sentence because the need to instantiate the object is, I think, made clear both in the next sentence ("To create a dataset...") and in the code examples.

Reply:
I don't think this is true. In fact, when I use the proposed path to the method, the link won't even build The delete method does appear to live in the project. https://github.com/CitrineInformatics/citrine-python/blob/49d7c8241d2c4a81438139e1d2483d8d99cc3b86/src/citrine/resources/project.py#L524

The delete method available within the dataset module is only for data objects     https://github.com/CitrineInformatics/citrine-python/blob/49d7c8241d2c4a81438139e1d2483d8d99cc3b86/src/citrine/resources/dataset.py#L281

Reply:
There are a few delete methods here to consider.
On the [dataset resource](https://github.com/CitrineInformatics/citrine-python/blob/49d7c8241d2c4a81438139e1d2483d8d99cc3b86/src/citrine/resources/dataset.py#L281), `dataset.delete()`, this deletes a data object, and as you pointed out, that's not what we want to use here.
On the [project collection](https://github.com/CitrineInformatics/citrine-python/blob/49d7c8241d2c4a81438139e1d2483d8d99cc3b86/src/citrine/resources/project.py#L524), `citrine.projects.delete()`. That takes a project id and deletes a project.
On the [dataset collection](https://github.com/CitrineInformatics/citrine-python/blob/49d7c8241d2c4a81438139e1d2483d8d99cc3b86/src/citrine/_rest/collection.py#L113), `project.datasets.delete()`. You're right that `delete` is not explicitly defined for `DatasetCollection`, so it uses the generic `Collection` delete method. This takes a dataset id and deletes the dataset.

Reply:
So, the code example below is correct (at least that's how it should work; I haven't tried deleting a dataset myself). You're right that the link I proposed above will not work because `delete` is not overwritten in that `DatasetCollection`. We could either link to the generic method in `Collection`, or we could forego links altogether. These specific implementations of `delete` don't involve novel signatures, nor do they have information in their docstrings that is surprising or necessary for the reader of this document to see. The code below demonstrates how to use them. So I suggest the text be simply "using the ``register`` and ``delete`` methods."

Reply:
Thanks! That's super useful information!

Reply:
I made the suggested changes in my latest commit :)